chain/gen/slashfilter/slashsvc/slashservice.go

```diff
@@ -69,7 +70,11 @@ func SlashConsensus(ctx context.Context, a ConsensusSlasherApi, p string, from s
 		for block := range blocks {
 			otherBlock, extraBlock, fault, err := slashFilterMinedBlock(ctx, sf, a, block)
 			if err != nil {
-				log.Errorf("slash detector errored: %s", err)
+				if strings.Contains(err.Error(), "could not find") {
```

Comment:
there is a `ipld.IsNotFound(err)` that should work here, pull in `ipld "github.com/ipfs/go-ipld-format"` as an import

Reply:
That is convenient. Updated in https://github.com/filecoin-project/lotus/pull/12907/commits/3f82cfe9e4f4143e50c3cb430461e5b6e99a3a43